x-pack/plugins/apm/server/lib/helpers/create_es_client/create_apm_event_client/index.ts

```diff
@@ -46,6 +46,7 @@ export type APMEventESSearchRequest = Omit<ESSearchRequest, 'index'> & {
   };
   body: {
     size: number;
+    track_total_hits: boolean | number;
```

Comment:
wouldn't be possible here to just have it as optional property and just use `number` as type?

Reply:
That's mostly how it works today. It's optional and defaults to 10.000. However, having a default makes it too easy to forget to set so we'd rather have it required, thus forcing the developer to make a decision.